var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -46,7 +46,27 @@ class Kokkos(Package):
 
     variant('qthreads', default=False, description="enable Qthreads backend")
     variant('cuda', default=False, description="enable Cuda backend")
-    variant('openmp', default=True, description="enable OpenMP backend")
+    variant('openmp', default=False, description="enable OpenMP backend")
+
+    # host architecture variant
+    variant(
+        'host_arch',
+        default='None',
```

Comment:
I was suggesting to use `None`, not `'None'`. The former is the Python null type, the latter is just a string.

Reply:
Affirmative.